code/modules/mob/inventory.dm

```diff
@@ -239,6 +239,7 @@
 	I.plane = initial(I.plane)
 	if(newloc)
 		I.forceMove(newloc)
+		I.removed_from_inventory(src)
```

Comment:
Looks like seepage from the flamer

Reply:
It is. I needed it to tracking object, this stuff is not in the code